ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss

```diff
@@ -155,9 +155,7 @@
   max-width: 23rem;
   height: 16rem;
   padding: $space-1 $space-5;
-  margin-top: -15px;
-  margin-top: 20px;
-  margin-left: -9px;
+  margin: 10px;
```

Comment:
Can we use a spacing value here to ensure our spacing is consistent? I think $space-3 should be about equivalent here. Same comment to the other 10px below

Reply:
@jlhdxw updated to $space-2 as per design review with Terry it ligns up best to 10px thank you